pkg/doc/signature/verifier/public_key_verifier.go

```diff
@@ -228,6 +230,39 @@ const (
 	secp256k1KeySize = 32
 )
 
+// ECDSASignature is a struct holding r and s values of an ECDSA signature.
+// TODO remove ECDSASignature and asn1decode() when Verify() uses pkg.Crypto interface
+type ECDSASignature struct {
+	R, S *big.Int
+}
+
+// NewECDSASignature creates a new ecdsaSignature object.
+func NewECDSASignature(r, s *big.Int) *ECDSASignature {
+	return &ECDSASignature{R: r, S: s}
+}
+
+func asn1decode(b []byte) (*ECDSASignature, error) {
```

Comment:
I don't understand this change. Where does JWS say to do asn1decode?

Was this tested for interop?

Reply:
JWS does not specify what signature encoding to use @troyronda

Reply:
What library did you test this change against?

Reply:
Tink uses DER keys by default which uses Asn1 encoding. This works 100% with Tink.

For the old signatures to work, we should use custom Tink key templates to specific IEE_1363 signing keys. 

Tink uses DER by default and this works perfectly with public key marshalled/parsed using x509 package.